onnxruntime/core/providers/rocm/RoctracerLogger.h

```diff
@@ -0,0 +1,170 @@
+#pragma once
```

Comment:
Rename the file to roctracer_logger.h, and for roctracer_logger.cc and thread_util as well?

Reply:
Sure.  But now the names will diverge between ORT and Kineto.  Just need to remember that the clones have different names and make sure the updates still happen.

Reply:
I see, can you please add a comment on this (foreign files in Kineto).